### DISCUSSION TITLE
Kong ingress controller querying konnectivity-agent with error

### DISCUSSION THREAD

**Discussion by pluk55** [Kong/kong#13136] [Help]
Hi, I have couple k8s clusters on Azure with kong ingress deploy. Once controller up and running, starts to output log about failed request to an ip address. I have checked IPs and all of them relates to konnectivity-agent pods:

2024/06/02 09:12:19 http: TLS handshake error from 10.10.16.74:48288: EOF
2024/06/02 09:12:19 http: TLS handshake error from 10.10.16.74:48294: remote error: tls: bad certificate
2024/06/02 09:12:30 http: TLS handshake error from 10.10.16.28:55902: EOF
2024/06/02 09:12:19 http: TLS handshake error from 10.10.16.28:55902: remote error: tls: bad certificate

Could you advice why ingress querying these pods?

**Comment by chronolaw** [comment]
@randmonkey , could you take a look? thanks.
